### ISSUE TITLE
Unable to install/restore 'Steeltoe.Management.Endpoint....' 2.2.0 packages

### ISSUE THREAD

**Issue by alfusinigoj** [SteeltoeOSS/steeltoe#2]
Throwing Exception, Unable to resolve dependency 'OpenCensus'. Source(s) used: 'nuget',  'Microsoft Visual Studio Offline Packages'.

**Comment by TimHess** [comment]
This appears to be a side affect of the OpenCensus package having pre-release status, where Steeltoe does not, but possibly only affecting the Visual Studio Package Manager.

A workaround (at least until OpenCensus ships a GA release) is to use the package manager console with the switch `-includePreRelease` instead of the GUI

**Comment by mzakrzewski** [comment]
Yeah, this one is frustrating. I don't agree with releasing a component (as released) with a dependency on a pre-released component. I get that you want to offer cool stuff and don't want to be tied to others' release schedules, but, for me, this "juice wasn't worth the squeeze" for getting this approved for our release. I get it though... rock, meet hard place.

**Comment by jkonicki** [comment]
A little background on why we chose the Alpha release of OpenCensus.  Steeltoe wrote all that code, tested it, and verified it worked.  Then we donated it all to OpenCensus csharp project, which was released as the Alpha.  So we decided to use that library as we expected to see a GA in the near future.  We did not expect that OpenCensus would then become the OpenTelemetry project and we would still be waiting on a GA release.  As of now, we are stuck until we OpenTelemetry release a usable build.  When all is said and done, we moved too quickly on picking up that code and should have relied on our code base until a GA was completed from the other open source project. 
We are looking forward to a GA of OpenTelemetry and at that time we will make the necessary changes for this library.  
Thank you for your patience.
